kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/cloud/Cloudlet.java

```diff
@@ -79,7 +78,7 @@ public void unsetCloudService(CloudService cloudService) {
     //
     // ----------------------------------------------------------------
 
-    protected void activate(ComponentContext componentContext) {
```

Comment:
This is a breaking change to the Cloudlet API which is meant to be extended.
It requires a major change in org.eclipse.kura.cloud package an org.eclipse.kura.api bundle.